dlt/destinations/path_utils.py

```diff
@@ -286,3 +286,20 @@ def get_table_prefix_layout(
         )
 
     return prefix
+
+
+def get_file_format_and_compression(file_path: str) -> Tuple[str, bool]:
```

Comment:
Actually, now that I think of it, this function more or less does the same as `ParsedLoadJobFileName`, I think you could use that everywhere really...

Reply:
Or maybe I am mistaken?